### PR TITLE
[FIX] Use suspend_security on search on purchase order for users that have no access right on purchases.

### PR DESCRIPTION
**Pull request by adrienpeiffer**
An other solution for #190

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
cc/ @sbidoul @andrius-preimantas

**Comment by pedrobaeza** [comment]
I prefer this one, but please document in the README in the installation section that you need this module from OCA/server-tools.

**Comment by adrienpeiffer** [comment]
Thanks @pedrobaeza ! It's done.

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Comment by sbidoul** [comment]
:+1:

File: purchase_partial_invoicing/README.rst [hunk 1]
```diff
@@ -12,6 +12,8 @@ To install this module, you need to:
 
 * Click on install button
 
+This module depends on base_suspend_security module which can be found on apps.odoo.com or on the OCA/sever-tools github repository.
```
Comment (guewen) [inline purchase_partial_invoicing/README.rst]: s/sever/server/

Comment (adrienpeiffer) [inline purchase_partial_invoicing/README.rst]: well spotted ! I'll change

**Comment by sbidoul** [comment]
> Is suspend_security() propagated to related objects like purchase.order.line? If not then po_line.invoiced = False would still be a problem :-)

Yes it is propagated as the context is attached to the recordset resulting from the search.

**Comment by adrienpeiffer** [comment]
@guewen it's done ! Thanks !

**Comment by yvaucher** [comment]
Some tests are failing and are related to this module: https://travis-ci.org/OCA/purchase-workflow/jobs/81918022#L322

`ValueError: "'purchase.order' object has no attribute 'suspend_security'" while evaluating`

in 
`openerp.addons.purchase_partial_invoicing.tests.test_purchase_partial_invoicing`

**Comment by adrienpeiffer** [comment]
@yvaucher Thanks ! It's green !

**Comment by yvaucher** [comment]
:+1:  merging...

**Pull request merged**